applications/ParticleMechanicsApplication/tests/beam_tests/hyperelastic_cantilever_beam/penalty_self_weight_load_2D_quad_test_parameters.json

```diff
@@ -56,17 +56,18 @@
             "Parameters"    : {
                 "model_part_name" : "Background_Grid.PENALTY_Condition_Auto1",
                 "particles_per_condition" : 7,
-                "penalty_factor"  : 1e15,
+                "penalty_factor"  : 1e20,
                 "constrained"     : "fixed"
             }
-        },{
+        },
+        {
             "python_module"   : "from_json_check_result_process",
             "kratos_module"   : "KratosMultiphysics",
             "help"                  : "",
             "process_name"          : "FromJsonCheckResultProcess",
             "Parameters"            : {
                 "check_variables"  : ["DISPLACEMENT"],
-                "input_file_name"  : "beam_tests/hyperelastic_cantilever_beam/self_weight_load_2D_quad_test_results.json",
+                "input_file_name"  : "beam_tests/hyperelastic_cantilever_beam/self_weight_load_2D_quad_test_boundary_particles_results.json",
```

Comment:
Why you change this? My idea was to make the penalty BC comparable to the original (node conforming) results. Can you revert and make all your boundary particle test to the original one?

Reply:
It only was the correct result compared to the node conforming BC because there was the error in the small cut ( for this special example where the shape function is either zero or 1 it is ok like that but that is not the general case). But if the small cut is considered you get slightly different results, as the background element containing the boundary particles is "blocked" completely. That's why I added a new result file.